### DISCUSSION TITLE
Help us to get more attention for this cool feature (Add custom G-code / M600 (color print) using the horizontal slider)

### DISCUSSION THREAD

**Discussion by Bumblebiber** [supermerill/SuperSlicer#3663] [General]
Go guys, make some noise. We need this :D 

https://github.com/prusa3d/PrusaSlicer/issues/5195

It would be super easy to implement for them. If some more people demand it, they might do it.

**Comment by jerry171** [comment]
I just wanted to say that I  would greatly appreciate this feature
